src/libraries/System.Console/src/System/ConsolePal.Unix.cs

```diff
@@ -1210,19 +1219,11 @@ private void AddKey(TermInfo.Database db, string extendedName, ConsoleKey key, b
         /// <returns>The number of bytes read, or a negative value if there's an error.</returns>
         internal static unsafe int Read(SafeFileHandle fd, byte[] buffer, int offset, int count)
         {
-            Interop.Sys.InitializeConsoleBeforeRead(convertCrToNl: true);
-            try
-            {
-                fixed (byte* bufPtr = buffer)
-                {
-                    int result = Interop.CheckIo(Interop.Sys.Read(fd, (byte*)bufPtr + offset, count));
-                    Debug.Assert(result <= count);
-                    return result;
-                }
-            }
-            finally
+            fixed (byte* bufPtr = buffer)
```

Comment:
@tmds, after this goes in, you can change back https://github.com/dotnet/runtime/pull/42303/files#diff-142ed2172d119029385e482ff7377130R267, right?

Reply:
Yes, I've just pushed a commit to that PR to remove the `TermInfo.cs` changes.